authors/apps/articles/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+from authors.apps.authentication.models import User
+from django.utils.text import slugify
+
+
+class ArticlesModel(models.Model):
+    """ This class defines the model for creating articles"""
+    slug = models.SlugField(db_index=True, max_length=128, unique=True, blank=True)
+    title = models.CharField(max_length=128, blank=False)
+    description = models.CharField(max_length=120, blank=False)
+    body = models.TextField(blank=False)
+    image_url = models.URLField(blank=True, null=True)
+    createdAt = models.DateTimeField(auto_now_add=True, auto_now=False)
```

Comment:
Could we have this as `created_at` and `updated_at` for uniformity with internal Django timestamp fields?